docs/source/development/reference/functions.rst

```diff
@@ -153,6 +153,18 @@ paragraph for more details.
     :type `area`: PJ_AREA
     :returns: :c:type:`PJ*`
 
+.. c:function:: PJ* proj_create_crs_to_crs_from_pj(PJ_CONTEXT *ctx, PJ *source_crs, PJ *target_crs, PJ_AREA *area, const char* const *options)
+
+    .. versionadded:: 6.2.0
+
+    Create a transformation object that is a pipeline between two known
+    coordinate reference systems.
+
+    This is the same as :c:func:`proj_create_crs_to_crs` except that the source and
+    target CRS are passed as PJ* objects.
```

Comment:
The `PJ` objects has to be the kind where `proj_is_crs()` returns true, right?

Reply:
yes, I hope this is clear enough from the argument name and above text.

Reply:
It is reasonable clear, but it can't hurt to explicitly state that the PJ has to be of the CRS variety.

Reply:
OK, precision just added